metal/resource_metal_device_network_type.go

```diff
@@ -20,12 +20,14 @@ func resourceMetalDeviceNetworkType() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"device_id": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Description: "The ID of the device on which the network type should be set",
+				Required:    true,
+				ForceNew:    true,
 			},
 			"type": {
 				Type:         schema.TypeString,
+				Description:  "Network type to set. Must be one of layer3, hybrid, layer2-individual and layer2-bonded",
```

Comment:
Perhaps in the future, we can define lists like this as a const and then use a function to strings.Join (with "Must be one of ... or ...") for Description while reusing the const for ValidateFunc.

Reply:
I can do it here and now.